### PR TITLE
Adds a chemical to the chem kit

### PR DESCRIPTION
**Pull request by MrMelbert**
## About The Pull Request

Adds Syndol to the chemical kit

- It deals minor liver damage
- It is a very addictive hallucinogen
- When a security officer is exposed, they will hallucinate other mobs as Nuke Ops
- When an assistant or clown is exposed, they will hallucinate other mobs as Security Officers
- When exposed to less than ten units, these unique hallucinations will only apply to all mobs *outside* of the view of the victim. 
- When exposed to more than ten units (an overdose), it applies to *all* mobs, not just those in view.
- All other members of the crew will just get normal hallucinations.
- Exposure is completely silent (until the liver damage kicks in, which takes a while)

image

## Why It's Good For The Game

Someone dropped the idea in Discord and I thought it was pretty funny. It also gives syndies a way to temporarily disorient a security officer a la the old chaos holoparasite  (oh god who was I chasing everyone is a syndie?).

## Changelog

:cl: Melbert
add: Adds Syndol to the chemical kit, an addictive hallucinogen that applies bonus effects when security officers, assistants, or clowns are exposed.
/:cl:

### CODE REVIEW

**Comment by Dawnseer** [comment]
Does a cargo liver exist yet? It should. And this should make cargo see people as IRS

**Comment by norsvenska** [comment]
> * When exposed to less than ten units, these unique hallucinations will only apply to all mobs _outside_ of the view of the victim.
> 
> * When exposed to more than ten units (an overdose), it applies to _all_ mobs, not just those in view.

wait, so does this mean that if you aren't overdosing, you'll only see the hallucinations through cameras and such? and if you overdose, it also affects those you can see with your eyes?

**Comment by kuricityy** [comment]
> > * When exposed to less than ten units, these unique hallucinations will only apply to all mobs _outside_ of the view of the victim.
> > * When exposed to more than ten units (an overdose), it applies to _all_ mobs, not just those in view.
> 
> wait, so does this mean that if you aren't overdosing, you'll only see the hallucinations through cameras and such? and if you overdose, it also affects those you can see with your eyes?

im assuming they mean it will only affect people that arent in view when they get injected, otherwise It would be pretty useless because youd know youre overdosing.

**Comment by norsvenska** [comment]
that makes so much more sense

**Comment by carlarctg** [comment]
the fact that you can only inject this poison the same way you inject much more efficient instant death mixes feels pretty unfortunate but idk how you'd fix that

same with the only way to procure it being a bundle purchase

**Comment by ju45he** [comment]
The new fear toxin from scarecrow be mad fire yo

File: code/modules/reagents/chemistry/reagents/drug_reagents.dm [hunk 1]
```diff
@@ -865,3 +865,54 @@
...
+/datum/reagent/drug/syndol
+	name = "Syndol"
+	description = "A potent and addictive hallucinogen used by syndicate agents disorient certain targets. \
+		It is said that the hallucinations it causes are tailored to the user's fears, but tests have been inconclusive, \
+		with subjects in security and assistants reporting wildly different experiences."
+	color = "#c90000"
+	taste_description = "metallic"
+	ph = 7
+	overdose_threshold = 10
+	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED
```
Comment (SyncIt21) [inline code/modules/reagents/chemistry/reagents/drug_reagents.dm:878]: Nice but how can it be synthesized? I see you didn't add any chemical reaction to create this recipe.

You also didn't add under what categories this reagent should show up in the chem dispenser UI

Comment (norsvenska) [reply code/modules/reagents/chemistry/reagents/drug_reagents.dm:878]: that flag just means that things like bees and the odysseus syringe thingy are able to make more of it

**Comment by SmArtKar** [comment]
It'd be extremely funny if it picked a random secoff's name instead of generic Security when creating delusions

File: code/modules/reagents/chemistry/reagents/drug_reagents.dm (hunk 1 above)
Comment (MrMelbert) [reply code/modules/reagents/chemistry/reagents/drug_reagents.dm:878]: yes, also it can appear in strange seed pool

**Comment by Loafin34** [comment]
this is so minor it should be a chem recipe. syndicate bomb + mindbreaker + fentanyl?

**Comment by MrMelbert** [comment]
I'd rather not

**Comment by MrMelbert** [comment]
> It'd be extremely funny if it picked a random secoff's name instead of generic Security when creating delusions

I'd do it if it actually took the appearance of in-round seccies but it just draws generic ones, so I CBA

**Pull request merged**